### ISSUE TITLE
Missing keys after v0.0.12 upgrade

### ISSUE THREAD

**Issue by FlorentFlament** [pivotalservices/tile-config-generator#16]
We have a script running on the "a9s PostgreSQL for PCF" tile with the following command:
```
tile-config-generator generate \
        --pivotal-file-path "${file_path}" \
        --base-directory "${config_dir}" \
        --do-not-include-product-version \
        --include-errands
```

With version v0.0.12 the generated `product.yml` is missing the following path `/product-properties/.properties.backup_config`. This was present when the file was generated with tile-config-generator v0.0.11. This is a mandatory field, and when we run the `om configure-product` command, we get the following error message:

```
Archive:  ./pivnet-product/a9s-postgresql-2.0.2.pivotal
	
  inflating: metadata/metadata.yml   
	
Expected to find a map key '.properties.backup_config' for path '/product-properties/.properties.backup_config' (found map keys: '.properties.plan_config_acknowledgement', '.properties.service_network_az', '.properties.syslog_selector', '.properties.syslog_selector.enabled.address', '.properties.syslog_selector.enabled.port', '.properties.syslog_selector.enabled.syslog_format', '.properties.syslog_selector.enabled.syslog_tls', '.properties.syslog_selector.enabled.syslog_transport')
	
Exit code 1
```

**Comment by calebwashburn** [comment]
This has been moved into an operations file to apply the value.

```features/backup_config-backup_on_azure.yml```

or 

```features/backup_config-backup_on_s3.yml```

or to disable 

```features/backup_config-backup_disabled.yml```

**Comment by bruxelles86** [comment]
Hi. I'm working with @FlorentFlament on this. We were already using the `backup_config-backup_disabled.yml` ops file (it looks unchanged?), but the problem seems to be that this file _replaces_ the value of `/product-properties/.properties.backup_config`. However, the `product.yml` being generated for us no longer has a `/product-properties/.properties.backup_config` key (and therefore there's nothing to be replaced). This is what that section now looks like:

```
product-properties:
  .properties.plan_config_acknowledgement:
    value:
    - acknowledge
  .properties.service_network_az:
    value: ((service_network_az))
  .properties.syslog_selector:
    value: enabled
  .properties.syslog_selector.enabled.address:
    value: ((syslog_selector/enabled/address))
  .properties.syslog_selector.enabled.port:
    value: ((syslog_selector/enabled/port))
  .properties.syslog_selector.enabled.syslog_format:
    value: ((syslog_selector/enabled/syslog_format))
  .properties.syslog_selector.enabled.syslog_tls:
    value: ((syslog_selector/enabled/syslog_tls))
  .properties.syslog_selector.enabled.syslog_transport:
    value: ((syslog_selector/enabled/syslog_transport))
```

**Comment by bruxelles86** [comment]
Just a correction from his original post: 

The error doesn't arise when we run `om configure-product` - it's actually when we use `bosh interpolate` to fill in variables in the `product.yml` file. It's at that stage that we get an error, because `bosh` is trying to apply an ops file that tells it to _replace_ the value of a key that doesn't exist.

Apologies for not having made that clear.

**Comment by calebwashburn** [comment]
Makes sense.  I’ll get this addressed.

**Comment by bruxelles86** [comment]
Awesome. Thanks! :)

**Comment by bruxelles86** [comment]
One _final_ clarification. The last version we were using before this error appeared was 0.0.9.

**Comment by calebwashburn** [comment]
Should be resolved with 0.0.13 - https://github.com/pivotalservices/tile-config-generator/releases/tag/v0.0.13 feel free to re-open or post another issue if something else isn't working.
